### DISCUSSION TITLE
Will minio move object files into subdirectory when there were millions of files in the directory it used for data storage?

### DISCUSSION THREAD

**Discussion by ZPC2048** [minio/minio#20433] [Q&A] [answered]
As metioned in title, I would like to know if minio has the ability to create directory hierarchy when there were too many files in the same directory. I have reached the subdirectory count limit when developing program on ext4, and I know that it is not recommended to use minion on ext4, so I just want to know if minio has that ability or not. Thank you for your help! ORZ.

**Comment by harshavardhana** [comment, marked as answer]
No

**Reply by ZPC2048** [reply]
Thanks!
